projects/sandbox/caliptra.md

```diff
@@ -0,0 +1,48 @@
+# Project application: **Caliptra**
+
+## Application
+
+### Sandbox application
+
+* Project name: Caliptra
+* Project repo(s):
+
+| Repo | What |
+| ---- | ---- |
+| [Primary Repo](https://github.com/chipsalliance/caliptra)        | This repo, with admin boilerplate and docs |
+| [Caliptra RTL](https://github.com/chipsalliance/caliptra-rtl)    | The primary repository with immutable RTL |
+| [Caliptra SW](https://github.com/chipsalliance/caliptra-sw)      | Caliptra software (ROM, FMC, runtime firmware), and libraries/tools needed to build and test |
+| [Caliptra U-Reg](https://github.com/chipsalliance/caliptra-ureg) | General-purpose libraries and tools for manipulating MMIO registers from Rust |
+| [Caliptra DPE](https://github.com/chipsalliance/caliptra-dpe)    | An implementation of a TCG DICE Protection Environment profile |
+* Brief summary of the project: The Caliptra project focuses on development of HW and SW IP for [The Caliptra Root of Trust](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md).
+* Project's open source license: **ASL 2.0**
+* Link to issue tracker: **TBD**
+* Link to website: **TBD**
+* Links to social media accounts: **N/A**
+* Who uses this project, and at what scale:
+* Why does the project want to join CHIPS Alliance:
+* Primary contact from the project during the Sandbox application process:
+  * Name: Matt Cockrell
+  * Email: mcockrell@google.com
+  * GitHub handle: mcockrell-google
+  * Role within the project: IC
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
+* Link to logo in .svg format
```

Comment:
Yeah I don't have that. Is this a hard need?

Reply:
We will use png